### PR TITLE
feat: add cli

### PR DESCRIPTION
**Pull request by xbcsmith**
Add a CLI to the project

### CODE REVIEW

File: cli/cmd/event/create.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+	if noindent {
+		fmt.Printf("%s\n", content)
+		return nil
+	}
+
+	if noindent {
+		fmt.Printf("%s\n", content)
+		return nil
+	}
```
Comment (ninjamast3r) [inline cli/cmd/event/create.go:79]: Duplicate statements

**Review by ninjamast3r — APPROVED** [review]
Good enough.

**Review by ninjamast3r — CHANGES_REQUESTED** [review]
fix the duplicate if

File: cli/cmd/event/create.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+func runCreateEvent(_ *cobra.Command, _ []string) error {
+	url := viper.GetString("url")
+	c, err := common.GetClient(url)
+	if err != nil {
+		return err
+	}
+
+	name := viper.GetString("name")
```
[the remark this answers is not in the document]
Comment (xbcsmith) [reply cli/cmd/event/create.go:33]: Will fix in issue #81

**Review by ninjamast3r — APPROVED** [review]

**Pull request merged**